string prefix_from
uint32 key_from
string prefix_to
uint32 key_to
# whether the user has confirmed this request
bool confirmed
---
bool success
bool confirm
